docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1995 @@
+{
```

Comment:
<p>I don't know. This function seems too trivial to be in a function. We'll end up splitting non-normalized methods anyways.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>

Reply:
Exactly. It is toooo trivial. I wanted to remove it but then I just left it there to see how you feel and if you want to add `normalize` and `p` to it. I will remove it in next set of commits.